Linters failing even on empty commit

##### SUMMARY

Linters failing with PR even on empty commit

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
- ci
- plugins/connection/kubectl.py

##### ANSIBLE VERSION
N/A

##### COLLECTION VERSION
Current `main` branch, most probably `stable-3` and `stable-5`

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
GitHub Action workers

##### STEPS TO REPRODUCE

Empty commit 259261b304db5afba182bd2edce289ef7028c830 pushed that triggered CI

##### EXPECTED RESULTS

All tests completed sucesfully


##### ACTUAL RESULTS

Linters failed: https://github.com/ansible-collections/kubernetes.core/actions/runs/13110109779/job/36571972999?pr=873
```
WARNING  Found incompatible custom yamllint configuration (.yamllint), please either remove the file or edit it to comply with:
  - comments.min-spaces-from-content must be 1
  - comments-indentation must be false
  - octal-values.forbid-implicit-octal must be true
  - octal-values.forbid-explicit-octal must be true.

Read https://ansible.readthedocs.io/projects/lint/rules/yaml/ for more details regarding why we have these requirements. Fix mode will not be available.
WARNING  Listing 2 violation(s) marked as ignored, likely already known
WARNING  Listing 1 violation(s) that are fatal
Error: Variables names must not be Ansible reserved names. (namespace)
Warning: Commands should not change things if nothing needs doing.
Warning: Commands should not change things if nothing needs doing.
Read documentation for instructions on how to ignore specific rule violations.

# Rule Violation Summary

  1 var-naming profile:basic tags:idiom

Failed: 1 failure(s), 2 warning(s) on 143 files. Profile 'production' was required, but 'min' profile passed.
no-changed-when: Commands should not change things if nothing needs doing. (warning) # ignored
plugins/connection/kubectl.py:184 Task/Handler: Run a command in My App pod

no-changed-when: Commands should not change things if nothing needs doing. (warning) # ignored
plugins/connection/kubectl.py:184 Task/Handler: Run a command in a pod

var-naming[no-reserved]: Variables names must not be Ansible reserved names. (namespace)
plugins/connection/kubectl.py:46

Command failed: got '2', expected '0'
Error: Process completed with exit code 1.
```

--- a/.ansible-lint-ignore
+++ b/.ansible-lint-ignore
@@ -1,3 +1,4 @@
+# https://docs.ansible.com/ansible-lint/docs/rules/
 # no-changed-when is not requried for examples
 plugins/connection/kubectl.py no-changed-when
 # false positive result

--- a/.config/ansible-lint-ignore.txt
+++ b/.config/ansible-lint-ignore.txt
@@ -1,4 +1,0 @@
-# no-changed-when is not requried for examples
-plugins/connection/kubectl.py no-changed-when
-# false positive result
-plugins/connection/kubectl.py var-naming[no-reserved]

--- a/.gitignore
+++ b/.gitignore
@@ -13,6 +13,7 @@ changelogs/.plugin-cache.yaml
 tests/output
 tests/integration/cloud-config-*
 .cache
+.ansible
 
 # Helm charts
 tests/integration/*-chart-*.tgz

--- a/.yamllint
+++ b/.yamllint
@@ -5,16 +5,24 @@ rules:
   braces:
     max-spaces-inside: 1
     level: error
+
   brackets:
     max-spaces-inside: 1
     level: error
+  comments:
+    min-spaces-from-content: 1
+  comments-indentation: false
   document-start: disable
   line-length: disable
   truthy: disable
   indentation:
     spaces: 2
     indent-sequences: consistent
+  octal-values:
+    forbid-implicit-octal: true
+    forbid-explicit-octal: true
 ignore: |
   .cache
   .tox
+  .ansible
   tests/output

--- a/tests/integration/targets/k8s_manifest_url/tasks/main.yml
+++ b/tests/integration/targets/k8s_manifest_url/tasks/main.yml
@@ -23,7 +23,7 @@
         - name: Update directory permissions
           file:
             path: "{{ manifests_dir.path }}"
-            mode: 0755
+            mode: '0755'
 
         - name: Create manifests files
           copy:

--- a/tests/integration/targets/lookup_kustomize/tasks/main.yml
+++ b/tests/integration/targets/lookup_kustomize/tasks/main.yml
@@ -45,7 +45,7 @@
     - name: make script as executable
       file:
         path: "{{ tmp_dir_path }}/install_kustomize.sh"
-        mode: 0755
+        mode: '0755'
 
     - name: Install kustomize
       command: "{{ tmp_dir_path }}/install_kustomize.sh"

